### PR TITLE
Add Taylor models GSoC project

### PR DESCRIPTION
**Pull request by dpsanders**

### CODE REVIEW

**Comment by mforets** [comment]
That's awesome! You beat me to it, i was planning to propose some reachability-related projects soon. Count me in as co-mentor if you wish.

> **Expected results**: A state-of-the-art library for Taylor models and guaranteed ODE solving.

IMHO that is too optimistic for one GSoC :) (and i'm an optimistic guy). Maybe remove the part "... and guaranteed ODE solving"?

One interesting GSoC project would be to reformulate `TaylorSeries` and `TaylorModels` to use the generic `MultivariatePolynomials` interface, so we can exploit sparse representations, static polynomials for fast evaluation, etc.

File: soc/projects/numerics.md [hunk 1]
```diff
@@ -66,24 +68,6 @@ This project proposal is for implementing native Julia algorithms for massively
 
 **Mentors:** [Sacha Verweij](https://github.com/Sacha0)
 
-## Generic linear algebra
-
-Julia supports many different numeric types, both in Base (e.g. `Float32`, `Float64`, `BigFloat`, `Complex`, `Rational`), as well as other packages (e.g. [Quaternions.jl](https://github.com/JuliaGeometry/Quaternions.jl), [ArbFloats.jl](https://github.com/JuliaArbTypes/ArbFloats.jl), [FixedPointNumbers.jl](https://github.com/JuliaMath/FixedPointNumbers.jl), [Unitful.jl](https://github.com/ajkeller34/Unitful.jl)).
```
Comment (ChrisRackauckas) [inline soc/projects/numerics.md]: is this intentional?

Comment (dpsanders) [reply soc/projects/numerics.md]: Thanks for pointing that out. I have repaired the damage in the new commit.

**Comment by dpsanders** [comment]
Bump.

**Pull request merged**